crates/recursor/src/recursor_dns_handle.rs

```diff
@@ -92,9 +92,48 @@ impl RecursorDnsHandle {
         let mut zone = zone;
         let mut ns = None;
 
-        // max number of forwarding processes
+        // Recursively search for authoritative name servers for the queried record to build an NS
```

Comment:
I'd like this PR to have a separate commit that adds this comment before the other changes, and then amends it when the other changes come in (and please squash the clippy fix commit).

Reply:
This is done.